### PR TITLE
stability improvements

### PR DESCRIPTION
**Pull request by fvzeppelin**
my NEO-7M likes to send this, so I need to catch it ;)

### CODE REVIEW

**Comment by dherrada** [comment]
Run `black . --target-version=py35` in this project's directory. You may need to install black with pip if you haven't already.

File: adafruit_gps.py [hunk 1]
```diff
@@ -45,7 +45,7 @@
 import time
 from micropython import const
 
-__version__ = "0.0.0-auto.0"
+__version__ = "3.6.3"
```
Comment (dherrada) [inline adafruit_gps.py:48]: Was there a reason this was done? I think we should have it set to the current version in github.

**Comment by dherrada** [comment]
@fvzeppelin Otherwise, seems good. I don't see any potential for any breakages arising from this, and I have tested it on the MTK3339, so I'll merge this when it's passing the CI

File: adafruit_gps.py (hunk 1 above)
Comment (fvzeppelin) [reply adafruit_gps.py:48]: No reason - accident

Comment (dherrada) [reply adafruit_gps.py:48]: no worries, just change it back after you run black. Thanks!

**Comment by fvzeppelin** [comment]
@dherrada now, checks have passed :) But I could not test it on the target, yet.

**Comment by dherrada** [comment]
@fvzeppelin No worries, black won't change anything functional. One more thing, change the version back to 3.6.3 and I can merge.

**Comment by fvzeppelin** [comment]
I had testes the version before applying black with a NEO-7M and an ItsyBitsy M0 Express.

**Comment by dherrada** [comment]
awesome. Merging

**Pull request merged**